packages/console/src/components/RadioGroup/Radio.module.scss

```diff
@@ -149,7 +149,7 @@
   }
 
   .content {
-    padding: _.unit(2) 0;
+    padding: calc(_.unit(2) - 1px) 0;
```

Comment:
Better check with the designer. Ideally, we should not have an odd padding number here. That is the idea of using a unit variable.

Reply:
Use height instead.